Automattic/jetpack
# E2E tests: save video for all tabs

#### Changes proposed in this Pull Request:

For tests that use multiple browser tabs we're not saving the video for all the opened pages. This should fix that.

#### Jetpack product discussion
pd5faL-1q-p2

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run Mailchimp test and make if fail -> there should be 2 video files saved in the output folder
